bugbug/repository.py

```diff
@@ -128,6 +137,8 @@ def _hg_log(revs):
             bug=rev[4],
             backedoutby=rev[5],
             author_email=rev[6],
+            files=rev[7].decode('utf-8').split('|'),
+            file_copies={fl[fl.find('(') + 1:-1]: fl[:fl.find('(') - 1] for fl in rev[8].decode('utf-8').split('|')} if len(rev[8]) > 0 else {},
```

Comment:
It'd be better to generate this with a loop earlier, so it is more readable, something along these lines (adding decode/adding b/removing b when necessary):
```Python
        file_copies = {}
        for file_copy in rev[8].split(b'|'):
            if not file_copy:
                continue

            parts = file_copy.split(b' (')
            copied = parts[0]
            orig = parts[1][:-1]
            file_copies[orig] = copied
```